### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation (?)

### DISCUSSION THREAD

**Discussion by raravena80** [kubeservice-stack/Community#2] [General]
Hello kubeservice-stack team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I believe you had an agenda item to present the project in our meeting. I'm following up, I think it would be great for you to present/discuss the project in the future.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by dongjiang1989** [comment]
@raravena80  thanks for U reply!  I would love to share kubeservice-stack

**Comment by raravena80** [comment]
Great @dongjiang1989 ! Let me know when you would like to present. Thank you!

**Reply by srust** [reply]
Hi @dongjiang1989! Just following up on this on behalf of TAG-Runtime - Please let us know any questions to myself (srust at srust.dev) or Ricardo, or in the #tag-runtime channel in the CNCF slack, or just add yourself to the [CNCF Tag-Runtime Agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) anytime.

Thank you!
